_studio/mfx_lib/decode/mjpeg/include/mfx_mjpeg_dec_decode.h

```diff
@@ -115,11 +115,11 @@ class VideoDECODEMJPEGBase_HW : public VideoDECODEMJPEGBase
 
 
 protected:
-    VideoVppJpegD3D9 *m_pCc;
-    // True if we need special VPP color conversion after decoding
-    bool   m_needVpp;
+    std::unique_ptr<VideoVppJpegD3D9> m_pCc;
```

Comment:
I still think it should be part of `mfx_UMC_FrameAllocator_D3D_Converter`

Reply:
Done.